app/src/main/kotlin/at/bitfire/davdroid/sync/Syncer.kt

```diff
@@ -242,26 +249,31 @@ abstract class Syncer<StoreType: LocalDataStore<CollectionType>, CollectionType:
      */
     operator fun invoke() {
         logger.log(Level.INFO, "${dataStore.authority} sync of $account initiated", extras.joinToString(", "))
+        var securityExceptionOccurred = false
 
         try {
             dataStore.acquireContentProvider()
         } catch (e: SecurityException) {
             logger.log(Level.WARNING, "Missing permissions for content provider authority ${dataStore.authority}", e)
+            securityExceptionOccurred = true
```

Comment:
If I don't overlook anything, we can just `return` here and don't need the `securityExceptionOccurred` then.

Reply:
Good catch :+1: